CRM/Contact/BAO/Query.php

```diff
@@ -5907,8 +5905,8 @@ public function optionValueQuery(
         // Special handling for on_hold, so that we actually use the 'where'
         // property in order to limit the query by the on_hold status of the email,
         // instead of using email.id which would be nonsensical.
-        if ($field['name'] == 'on_hold') {
-          $wc = "{$field['where']}";
+        if ($field['name'] === 'on_hold') {
+          $wc = $field['where'];
```

Comment:
@eileenmcnaughton any reason for the removal of the quotes here?

Reply:
@seamuslee001 readability